docs/puzzles/pooling.md

```diff
@@ -0,0 +1,320 @@
+---
+id: pooling
+title: Pooling
+---
+
+The way that Chia Network does pooling is unlike any blockchain that has come before it. Pool operators actually rely on an on-chain smart contract to verify that they will be able to directly claim any potential pool rewards that farmers create.  This allows pools to trust farmers enough to pay them out, while still keeping the power of making the blocks in the hands of the farmer.  This means that the decentralization of the network remains the same, even as the rewards get concentrated to the pool!
+
+In this section, we're going to break down how all of this works in Chialisp.  This section assumes you have already read the section about [singletons](https://chialisp.com/docs/puzzles/singletons) (or at least understand how they work) as that is the outer contract that wraps the pooling puzzles.
+
+## Design Requirements
+
+There are a few requirements that were set for how the pooling protocol would work on the Chia Network. Let's go over them now:
+
+* **The farmer still farms the blocks.** This is incredibly important for network decentralization. If this is not true, then the bigger a pool gets, the closer it is to gaining 51% of the network resources.  We still want the farmers to create the blocks, but we need some way to assure the pool that the farmer will give them the reward when they do.  We do this by creating plots that farm directly to a specific puzzle hash, and ensuring that puzzle hash is something that pool can claim.
```

Comment:
`The farmer still farms the blocks` ->
The farmer farms the blocks, not the pool.